script/lint.js

```diff
@@ -136,12 +137,54 @@ const LINTERS = [ {
       process.exit(1)
     }
   }
+}, {
+  key: 'patches',
+  roots: ['patches'],
+  test: () => true,
+  run: () => {
+    const patchesDir = path.resolve(__dirname, '../patches')
+    for (const patchTarget of fs.readdirSync(patchesDir)) {
+      const targetDir = path.resolve(patchesDir, patchTarget)
+      // If the config does not exist that is OK, we just skip this dir
+      const targetConfig = path.resolve(targetDir, 'config.json')
+      if (!fs.existsSync(targetConfig)) continue
+
+      const config = JSON.parse(fs.readFileSync(targetConfig, 'utf8'))
+      for (const key of Object.keys(config)) {
+        // The directory the config points to should exist
+        const targetPatchesDir = path.resolve(__dirname, '../../..', key)
+        expect(fs.existsSync(targetPatchesDir)).to.equal(true, `target patch directory: "${targetPatchesDir}" should exist`)
+        // We need a .patches file
+        const dotPatchesPath = path.resolve(targetPatchesDir, '.patches')
+        expect(fs.existsSync(dotPatchesPath)).to.equal(true, `.patches file: "${dotPatchesPath}" should exist`)
```

Comment:
super minor nit: i think these paths will be printed relative to this script? ideally they'd be relative to either `src/` or `src/electron`

Reply:
These paths will be printed as absolute paths currently